src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/bespoke/YieldPerformanceReportITest.java

```diff
@@ -0,0 +1,330 @@
+package uk.gov.hmcts.juror.api.moj.report.bespoke;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.juror.api.AbstractControllerIntegrationTest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.YieldPerformanceReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.AbstractReportResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.YieldPerformanceReportResponse;
+import uk.gov.hmcts.juror.api.moj.domain.UserType;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.Set;
+
+import static org.assertj.core.api.BDDAssertions.within;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@DisplayName("Controller: " + YieldPerformanceReportITest.URL)
+@Sql({
+    "/db/truncate.sql",
+    "/db/mod/truncate.sql",
+    "/db/mod/reports/YieldPerformanceReportITest_typical.sql",
+})
+@SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert"//False positive
+)
+class YieldPerformanceReportITest extends AbstractControllerIntegrationTest<YieldPerformanceReportRequest,
+    YieldPerformanceReportResponse> {
+    public static final String URL = "/api/v1/moj/reports/yield-performance";
+    public static final String LOCAL_DATE = "LocalDate";
+
+    @Autowired
+    public YieldPerformanceReportITest(TestRestTemplate template) {
+        super(HttpMethod.POST, template, HttpStatus.OK);
+    }
+
+    @Override
+    protected String getValidUrl() {
+        return URL;
+    }
+
+    @Override
+    protected String getValidJwt() {
+        return createJwt(
+            "test_bureau_standard",
+            "400",
+            UserType.BUREAU,
+            Set.of(),
+            "400"
+        );
+    }
+
+    @Override
+    protected YieldPerformanceReportRequest getValidPayload() {
+        return null;
```

Comment:
Should have a valid payload

Reply:
all the payloads are different so need to override this method